PhotDM.tex

```diff
@@ -107,7 +107,7 @@ \section*{Link to IVOA Architecture}
 \caption{PhotDM can be used to enhance and abstract SSAP and TAP access, in particular 
 to help on the automatic translation of magnitudes to fluxes and to add provenance 
 metadata to these magnitudes. Also it can be used to generate SEDs using the SpectralDM.
-PhotDM makes use of utypes from different IVOA data models, including CharDM and UCDs and
+PhotDM makes use of Utypes from different IVOA data models, including CharDM and UCDs and
```

Comment:
Capitalisation of "utype" is rather inconsistent in the document.  If you change it here, why not make it consistent in the whole document?  My vote would be for consistent lowercase, though -- what would be the rationale for having it uppercase?  I don't think it's an actual proper name.